apps/amo/tests/__init__.py

```diff
@@ -332,7 +332,7 @@ def assertNoFormErrors(self, response):
 
     def assertLoginRedirects(self, response, to, status_code=302):
         # Not using urlparams, because that escapes the variables, which
-        # is good, but bad for assertRedirects which will fail.
+        # is good, but bad for assert3xx which will fail.
```

Comment:
I didn't sed the line below, it is currently using assert3xx, so the comment is now more correct.